float64[] center
float64[] central_axis
float64 radius
